assets/js/pages/ChecksSelection/hooks.test.jsx

```diff
@@ -0,0 +1,94 @@
+import { networkClient } from '@lib/network';
+import MockAdapter from 'axios-mock-adapter';
+import { faker } from '@faker-js/faker';
+import { act, renderHook } from '@testing-library/react';
+import { hookWrapperWithState } from '@lib/test-utils';
+import { selectableCheckFactory } from '@lib/test-utils/factories';
+import { useChecksSelection } from './hooks';
+
+const axiosMock = new MockAdapter(networkClient);
+
+describe('useChecksSelection', () => {
+  afterEach(() => {
+    axiosMock.reset();
+  });
+
+  it('should return default state', async () => {
+    let hookResult;
+    const [hookWrapper, _] = hookWrapperWithState();
+
+    await act(() => {
```

Comment:
Maybe ```await act(async () => {...```    to ensure the async function completes before making assertions

Reply:
Not sure whether it affects positively, we're not awaiting anything in those funcions.
In any case does not seem to affect negatively 😄 acf888acabb0dc201611af44d94e1623f3a47ffe